apps/www/registry/default/ui/command.tsx

```diff
@@ -37,17 +37,24 @@ const CommandDialog = ({ children, ...props }: CommandDialogProps) => {
   )
 }
 
+interface CommandInputProps
+  extends React.ComponentPropsWithoutRef<typeof CommandPrimitive.Input> {
+  inputClassName?: string
+}
 const CommandInput = React.forwardRef<
   React.ElementRef<typeof CommandPrimitive.Input>,
-  React.ComponentPropsWithoutRef<typeof CommandPrimitive.Input>
->(({ className, ...props }, ref) => (
-  <div className="flex items-center border-b px-3" cmdk-input-wrapper="">
+  CommandInputProps
+>(({ className, inputClassName, ...props }, ref) => (
+  <div
+    className={cn("flex items-center border-b px-3", className)}
+    cmdk-input-wrapper=""
+  >
     <Search className="mr-2 h-4 w-4 shrink-0 opacity-50" />
     <CommandPrimitive.Input
       ref={ref}
       className={cn(
         "flex h-11 w-full rounded-md bg-transparent py-3 text-sm outline-none placeholder:text-muted-foreground disabled:cursor-not-allowed disabled:opacity-50",
-        className
+        inputClassName
```

Comment:
@gulshanAI did you mean to add this to the PR? Can you confirm please?

Reply:
No do not add this 'apps/www/registry/default/ui/command.tsx'